src/redist/targets/GetRuntimeInformation.targets

```diff
@@ -24,9 +24,7 @@
     </PropertyGroup>
 
     <PropertyGroup>
-      <ProductMonikerRid Condition=" '$(Rid)' == 'ubuntu.16.04-x64' OR
-                                   '$(Rid)' == 'rhel.6-x64' OR
-                                   '$(Rid)' == 'linux-musl-x64' ">$(Rid)</ProductMonikerRid>
+      <ProductMonikerRid Condition="$(Rid.StartsWith('linux-musl'))">$(Rid)</ProductMonikerRid>
```

Comment:
RHEL 6 and Ubuntu 16.04 are unsupported operating systems in .NET 7: https://github.com/dotnet/core/blob/main/release-notes/7.0/supported-os.md#linux. Removed from the condition while I was at it.